ports/gecode/portfile.cmake

```
@@ -0,0 +1,25 @@
+vcpkg_from_github(
+    OUT_SOURCE_PATH SOURCE_PATH
+    REPO Gecode/gecode
+    REF 2925392ac7a0c8a7b33aaee51721d84725605919
+    SHA512 15b01bdb4c190557c96a257dbdb40b368310dae5665c568028570d52e7271daa79e9f27d981c5370b314784326af626a8c6ed1f53ce2c7db7b9e8d644d55fe97
+    HEAD_REF master
+)
+
+vcpkg_configure_cmake(
+    SOURCE_PATH ${SOURCE_PATH}
+    PREFER_NINJA
+)
+
+vcpkg_install_cmake()
```

```suggestion
vcpkg_cmake_install()
```
The sane as above, using new function for new ports.